easybuild/easyconfigs/k/Kraken2/Kraken2-2.0.7-beta-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PackedBinary'
+
+name = 'Kraken2'
+version = '2.0.7-beta'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.ccb.jhu.edu/software/%(namelower)s/'
+description = """Kraken is a system for assigning taxonomic labels to short DNA sequences,
+ usually obtained through metagenomic studies. Previous attempts by other
+ bioinformatics software to accomplish this task have often used sequence
+ alignment or machine learning techniques that were quite slow, leading to
+ the development of less sensitive but much faster abundance estimation
+ programs. Kraken aims to achieve high sensitivity and high speed by
+ utilizing exact alignments of k-mers and a novel classification algorithm."""
+
+# part is compiled with $CXX, the rest is in Perl
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'openmp': True, 'cstd': 'c++11'}
+
+github_account = 'DerrickWood'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_CXX-CXXFLAGS.patch']
+checksums = [
+    'baa160f5aef73327e1a79e6d1c54b64b2fcdaee0be31b456f7bc411d1897a744',  # v2.0.7-beta.tar.gz
+    '191291e51a846d193a12486ab84496d5dd221a05c84648688b1351bb84d7adb2',  # Kraken2-2.0.7-beta_CXX-CXXFLAGS.patch
+]
+
+dependencies = [
+    ('Perl', '5.28.0'),
+    ('BLAST+', '2.7.1'),
+    ('wget', '1.20.1', '-GCCcore-7.3.0', True),
```

Comment:
@SethosII Same here, this should just be `('wget', '1.20.1')`?

Reply:
@boegel I changed this.